tasks/firmware_installer.yml

```diff
@@ -39,5 +37,5 @@
     msg: "{{ details.stdout_lines }}"
   when: details.rc == 0
 
-- name: Firmware RPM cleanup
-  package: name={{ item }} state=absent
+- name: Firmware RPM cleanup {{ item }}
```

Comment:
I wanted to see the name of the package yum is trying to remove. Indeed there are multiple space in the name which actually problematic in ansible > 2.7.  Therefore just extracted the the name from the long string. Name is "firmware-ilo4.i386 : HPE Integrated Lights-Out 4 firmware" which is weird for yum.